__tests__/app.test.js

```diff
@@ -425,7 +426,25 @@ describe('Additional GET Requests', () => {
         });
     });
     describe('GET /api/articles (topic query)', () => {
-        
+        test('should allow the get articles endpoint to take a query of topic and filter by it', () => {
+            return request(app)
+                .get('/api/articles?topic=cats')
+                .expect(200)
+                .then(({ body }) => {
+                    const { articles } = body;
+                    articles.forEach((article) => {
+                        expect(article.topic).toBe('cats');
+                    })
+                })
+        });
+        test('should give 400 Bad Request if the topic category is not valid', () => {
+            return request(app)
+                .get('/api/articles?topic=cacti')
+                .expect(400)
+                .then(({ body }) => {
+                    expect(body.msg).toBe('Bad Request');
+                })
+        });
```

Comment:
We're just missing another test here- what would happen if it's a valid topic but has no articles?